modules/social_features/social_user/social_user.install

```diff
@@ -38,6 +38,14 @@ function social_user_update_8001(&$sandbox) {
 }
 
 /**
+ * Add view, block users permissions for the content,site managers.
+ */
+function social_user_update_8002() {
+  _social_user_set_permissions();
```

Comment:
We should just grant the new permissions here to the role that should have them (preferably based on the permission it's splitting off from if possible). Calling this function in an update hook might reset permissions that a site-administrator or developer has changed.

e.g. if a site using Open Social has revoked the "select account cancellation method" (which should probably not be given to all users anyway?) then this update hook will grant that permission again.

Reply:
Hi, @Kingdutch, yup we could update method for setting permissions. If you want - you could fix it by yourself, otherwise, I could fix it later.

Reply:
Just a friendly reminder that this still needs a little bit of work ;)